lib/c-api/src/wasm_c_api/wasi/mod.rs

```diff
@@ -233,7 +233,7 @@ unsafe fn wasi_env_with_filesystem_inner(
     let module = &module.as_ref()?.inner;
     let imports = imports?;
 
-    let (wasi_env, import_object, runtime) = prepare_webc_env(
+    let (wasi_env, import_object) = prepare_webc_env(
```

Comment:
those changes are already part of master. You might need to rebase or merge from master.